src/gql/queries/failed-task-status-icon-tooltip.graphql

```diff
@@ -0,0 +1,14 @@
+query GetFailedTaskStatusIconTooltip($taskId: String!) {
+  task(taskId: $taskId) {
+    execution
+    id
+    displayName
+    timeTaken
+  }
```

Comment:
We shouldn't need to request this data a second time it can be fetched in the original query when tasks are fetched and passed down to the necessary component along side the status and task id which is already passed down.

https://github.com/evergreen-ci/spruce/blob/d61b948175721dd24d3f984eeec423af94700642/src/gql/queries/get-mainline-commits.graphql#L22-L27